Source/WebCore/css/StyleProperties.cpp

```diff
@@ -705,6 +707,58 @@ String StyleProperties::fontVariantValue() const
     return result.toString();
 }
 
+String StyleProperties::fontSynthesisValue() const
+{
+    StringBuilder result;
+
+    auto getExplicitLonghandValue = [&](CSSPropertyID propertyID) -> CSSValue* {
+        auto foundPropertyIndex = findPropertyIndex(propertyID);
+        if (foundPropertyIndex == -1)
+            return nullptr;
+
+        auto property = propertyAt(foundPropertyIndex);
+        if (property.isImplicit())
+            return nullptr;
+
+        return property.value();
+    };
+
+    auto appendWithPrefixIfNeeded = [&](const char* word) {
+        if (!result.isEmpty())
+            result.append(' ');
+        result.append(String(word, strlen(word)));
```

Comment:
`result.append(word)` once you switch to ASCIILiteral. No need to unnecessarily construct a String.